### ISSUE TITLE
ListFilter not working

### ISSUE THREAD

**Issue by marnickmenting** [vdhicts/cyberfusion-cluster-api-client#81]
## Describe the bug

I try to use the ListFilter, but it returns all results.

## Reproduction
```
$filter = ListFilter::forModel(new Certificate())->addFilter('commonNames', 'support.inserve.nl');
$cert = $cf->api->certificates()->list($filter)->getData('certificates');
```
## Expected behavior

Only return certificate with common_names = ['support.inserve.nl'];

## Actual behavior

It returns all certificates in the cluster.

## Additional information

Trying to update our code for latest changes on LetsEncrypt certificates.

Version 1.35.0

**Comment by dvdheiden** [comment]
Thank you for reporting. Will take a look at this asap.

**Comment by WilliamDEdwards** [comment]
Note that the property uses snake case ;)

**Comment by marnickmenting** [comment]
The ListFilter only accepts camelCase, I assume it converts under the hood.

**Comment by dvdheiden** [comment]
I switched to reflection to check which properties are allowed. That messed up the case. Will fix this today.

**Comment by dvdheiden** [comment]
Fixed in v1.36.0.

**Comment by marnickmenting** [comment]
Yes, it works now, thanks. For reference, this is the working code:

```
$filter = ListFilter::forModel(new Certificate())->addFilter('common_names', 'support.inserve.nl');
$cert = $cf->api->certificates()->list($filter)->getData('certificates');
```
